### PR TITLE
Add Nginx and Apache web servers close #18

### LINKED ISSUE #18: Add form section for web servers
Nginx. 
Apache

### PR DESCRIPTION
**Pull request by skafandri**

### CODE REVIEW

File: symfony/ServiceDefinition.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<?php
+declare(strict_types=1);
```
Comment (ibrahimgunduz34) [inline symfony/ServiceDefinition.php]: missing space on before and after equal sign.

**Comment by skafandri** [comment]
ping @danielmihalache1983

**Comment by danielmihalache1983** [comment]
@skafandri I`m trying to understand the changes here. I`m not sure I should? If you want me to approve it, let me know.
Overall I`m trying to get used with the application and get it running

**Comment by skafandri** [comment]
@danielmihalache1983 Did you manage to run the application locally? If not, then we must prioritize https://github.com/docker-symfony/docker-symfony/issues/33 which will enable us to run the application on a **public** production and staging/test environment. 

Should you understand all the changes? That's up to you. We are here to answer any questions.

**Comment by danielmihalache1983** [comment]
@skafandri I`m having issues running it localy because I am on windows 10.
I`m at this now:
image

maybe I can add an issue to also have deploy scripts for w10?

**Comment by skafandri** [comment]
I see now, the provided scripts are for Linux. As mentioned in README *contributions for other scripting commands are welcome*. That would be   a great contribution.

**Comment by skafandri** [comment]
Basically you need the equivalent of https://github.com/docker-symfony/docker-symfony/blob/master/deploy/build.sh on Windows

**Comment by skafandri** [comment]
You could take a look at https://www.google.ro/amp/s/www.howtogeek.com/249966/how-to-install-and-use-the-linux-bash-shell-on-windows-10/amp/, but I never tried it before.

**Comment by danielmihalache1983** [comment]
We can have a build.bat for windows. But the problem I see is here
https://github.com/docker-symfony/docker-symfony/blob/bcabfb9517a932a483a5c6a620cfd9e64f02bad8/deploy/Dockerfile#L18

Dockerfile is containing linux specific commands which can`t execute on windows, right?

**Review by danielmihalache1983 — APPROVED** [review]
Functionality wise, looks good to me.
web servers form section is there and it downloads docker-compose.yml according to user selection.

**Pull request merged**